src/db/mod.rs

```diff
@@ -0,0 +1,26 @@
+use std::env;
+
+use dotenv::dotenv;
+use tokio::sync::OnceCell;
+use tokio_postgres::{Client, Error, NoTls};
+
+static CLIENT: OnceCell<Client> = OnceCell::const_new();
+
+async fn init_client() -> Result<Client, Error> {
+    dotenv().ok();
+    let database_url = env::var("DATABASE_URL").expect("DATABASE_URL must be set");
```

Comment:
I've initialised a configuration module.
```suggestion
    let database_url = get_configuration().connection_string();
```
And cleanup the remaining code (like removing dotenv from cargo